src/overlays/actors/ovl_Bg_Market_Step/z_bg_market_step.c

```diff
@@ -7,20 +7,34 @@
 void BgMarketStep_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgMarketStep_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
-const ActorInit Bg_Market_Step_InitVars = {
-    ACTOR_BG_MARKET_STEP,
-    ACTORTYPE_BG,
-    FLAGS,
-    OBJECT_MARKET_OBJ,
-    sizeof(BgMarketStep),
-    (ActorFunc)BgMarketStep_Init,
-    (ActorFunc)func_800BDFB0,
-    (ActorFunc)func_800BDFB0,
-    (ActorFunc)BgMarketStep_Draw
+const ActorInit Bg_Market_Step_InitVars = { ACTOR_BG_MARKET_STEP,
+                                            ACTORTYPE_BG,
+                                            FLAGS,
+                                            OBJECT_MARKET_OBJ,
+                                            sizeof(BgMarketStep),
+                                            (ActorFunc)BgMarketStep_Init,
+                                            (ActorFunc)func_800BDFB0,
+                                            (ActorFunc)func_800BDFB0,
+                                            (ActorFunc)BgMarketStep_Draw };
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_F32(unkFC, 1, ICHAIN_CONTINUE),
+    ICHAIN_F32(unk100, 1, ICHAIN_CONTINUE),
+    ICHAIN_F32(unk104, 1, ICHAIN_CONTINUE),
+    ICHAIN_VEC3F_DIV1000(scale, 1000, ICHAIN_STOP),
 };
-*/
 
-GLOBAL_ASM("asm/non_matchings/ovl_Bg_Market_Step_0x80AF0060/BgMarketStep_Init.asm")
+s32 D_80AF0120[] = { 0x0601F050, 0x06018DA0 };
```

Comment:
This is really an array of pointers to to some object file data.
Can you try to determine the type? Likely Gfx*
The pointers should be defined in undefined syms.